changed.d

```diff
@@ -506,8 +506,8 @@ Please supply a bugzilla version
                 with(changelogStats)
                 {
                     auto changelog = changelogEntries > 0 ? "%d major change%s and".format(changelogEntries, changelogEntries > 1 ? "s" : "") : "";
-                    w.put("$(VER) comes with {changelogEntries} {bugzillaIssues} fixed Bugzilla issue{bugzillaIssuesPluaral}.
-                            A huge thanks goes to the {nrContributors} contributor{nrControbutorsPlural} who made $(VER) possible."
+                    w.put("$(VER) comes with {changelogEntries} {bugzillaIssues} fixed Bugzilla issue{bugzillaIssuesPlural}.
+                            A huge thanks goes to the {nrContributors} contributor{nrContributorsPlural} who made $(VER) possible."
```

Comment:
![image](https://user-images.githubusercontent.com/4370550/43003456-dfbf5a90-8c2c-11e8-9a74-f5afa5fdcbe6.png)

Hmm as the preview doesn't include the contributors stats (to avoid random changes), maybe we should change the plural check to `!= 0`?